bullet_train/app/views/account/onboarding/user_details/edit.html.erb

```diff
@@ -60,17 +60,16 @@
     // figure out the rails timezone value.
     var railsValue = jsTimezoneMapping[jstz.determine().name()];
 
-    var useBrowserTimeZone = $("#user_time_zone")[0].dataset.useBrowserTimeZone
+    var useBrowserTimeZone = document.querySelector("#user_time_zone").dataset.useBrowserTimeZone
 
     if (useBrowserTimeZone == "true") {
-      // set the form accordingly.
-      var $option = $("#user_time_zone option[value=\"" + railsValue + "\"]")
-      $option.prop('selected', true);
-
-      // update the select2 as well. is there a better way to handle this?
-      // why don't _they_ handle this for us?
-      $("#select2-user_time_zone-container").attr('title', $option.text());
-      $("#select2-user_time_zone-container").text($option.text());
```

Comment:
These lines don't seem to be necessary anymore. Setting the `value` attribute on the select element itself (`#user_time_zone`) now seems to make select2 automatically update.